packages/ui/src/components/SidePanel/SidePanel.tsx

```diff
@@ -106,11 +106,7 @@ const SidePanel = ({
 
   return (
     <Dialog.Root open={open} onOpenChange={handleOpenChange} defaultOpen={defaultOpen}>
-      {triggerElement && (
-        <Dialog.Trigger asChild className={__styles.trigger}>
-          {triggerElement}
-        </Dialog.Trigger>
-      )}
+      {triggerElement && <Dialog.Trigger asChild>{triggerElement}</Dialog.Trigger>}
```

Comment:
Not sure what the change here is for but seems to work fine in testing

Reply:
We had some old sass styling applied to the element that was breaking the child button's styling. Seems like this was the only place we use it, so remove the class.